config.js

```diff
@@ -51,6 +51,23 @@ config.server = {
     maxPageSize: '100'
```

Comment:
An entry in CHANGES_NEXT_RELEASE file regarding this PR should be included. Something like this:

```
- Add: CORS support (#500)
```